stdlib/public/core/Integers.swift

```diff
@@ -2317,6 +2317,59 @@ extension FixedWidthInteger {
     return byteSwapped
 #endif
   }
+  
+  // Default implementation of multipliedFullWidth.
+  //
+  // This implementation is mainly intended for [U]Int64 on 32b platforms. It
+  // will not be especially efficient for other types that do not provide their
+  // own implementation, but neither will it be catastrophically bad. It can
+  // surely be improved on even for Int64, but that is mostly an optimization
+  // problem; the basic algorithm here gives the compiler all the information
+  // that it needs to generate efficient code.
```

Comment:
The commentary is really nice, but shouldn't the documentation be placed above the function? I imagine you can put the commentary right above the docs.

Reply:
The documentation is provided by the protocol. It doesn't need to be on every implementation.

Reply:
Good to know!